cmd/kubeadm/app/discovery/file/file.go

```diff
@@ -106,6 +108,11 @@ func ValidateConfigInfo(config *clientcmdapi.Config, clustername string) (*clien
 		}
 		return true, nil
 	})
+	if err == wait.ErrWaitTimeout {
+		timeoutErr := errors.Errorf("Abort reading the %s ConfigMap after timeout of %v", bootstrapapi.ConfigMapClusterInfo, discoveryTimeout)
+		klog.V(1).Infof("[discovery] %v\n", timeoutErr)
+		return nil, timeoutErr
```

Comment:
should be enough to just:
```
return nil, errors.Errorf("Abort reading the %s ConfigMap after timeout of %v", bootstrapapi.ConfigMapClusterInfo, discoveryTimeout)
```
without the klog v1, otherwise for --v=1 the error will be printed twice.